CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
-add_definitions(-Wall -Werror -Wno-return-type-c-linkage)
+add_definitions(-Wall -Werror -Wno-return-type-c-linkage -Wno-sign-compare)
```

Comment:
Is there a reason you chose to disable this warning rather than fixing occurrences of it?

Reply:
There are a lot of instances of the warning and it would have made this diff quite noisy for a change that isn't directly related to the version bump. I've opened [an issue](https://github.com/runtimeverification/llvm-backend/issues/520) to go back and properly fix the problem.

If you'd like it done directly in this PR I can do that, but I don't think it's urgent.